### PR TITLE
Added not for the Optimized packager section.

### PR DESCRIPTION
**Pull request by SeqLaz**
Added note for the Optimized Packager section.

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3976 Enhance your project with the "Optimized Packager" add precision](https://app.clickup.com/t/2192114/QF-3976)

**Comment by suricactus** [comment]
It is correct. I think we are quite confident with the new packager, so we can switch to in the upcoming versions.

**Review by suricactus — APPROVED** [review]

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 1]
```diff
@@ -347,6 +347,9 @@ For the old (QGIS) offliner:
 
 !![](../../assets/images/qfc_offline_packager.png,700px)
 
+!!! note
+    This configuration must be set in the Settings section of each project in [QFieldCloud](https://app.qfield.cloud/).
```
Comment (suricactus) [inline documentation/get-started/tutorials/get-started-qfc.en.md:351]:
```suggestion
    This configuration must be set in the Settings page of each project in [QFieldCloud](https://app.qfield.cloud/).
```

Do we call these pages or sections? I usually use pages, but I am ok to call it a section if the documentation already uses this terminology.

Comment (SeqLaz) [reply documentation/get-started/tutorials/get-started-qfc.en.md]: I completely agree that we should use a page instead of a section.

**Comment by suricactus** [comment]
There are some linting errors before merging.

**Review by suricactus — APPROVED** [review]

**Pull request merged**